ext/js/data/options-util.js

```diff
@@ -1132,6 +1133,36 @@ export class OptionsUtil {
         }
     }
 
+    /**
+     * - Change 'viewNote' action to 'viewNotes'
+     * @type {import('options-util').UpdateFunction}
+     */
+    async _updateVersion25(options) {
+        for (const profile of options.profiles) {
+            /* eslint-disable @stylistic/no-multi-spaces */
```

Comment:
the linter is complaining about this not being re-enabled, but also, you should really only loop over all of the existing inputs, and just modify the ones where it was previously `viewNote` and change it to `viewNotes`.

Reply:
👍